ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokenPanel.qml

```diff
@@ -95,7 +95,7 @@ Item {
             model: root.tokensModel
 
             delegate: Text {
-                text: "name: " + name + ", descr: " + description + ", supply: " + supply
+                text: "name: " + name + ", descr: " + description + ", supply: " + supply + ", status: " + deployState
```

Comment:
is this some temporary text?

Reply:
Yes, this is just a mock to see results. This UI is hidden by a trigger in Advanced section.